start_x: 0.0288
start_y: 0.0680
end_x: -0.007758
end_y: 0.09518
theta_goal: 0

# Method
method: 2

# Visual servoing parameters
Tolerance: 0.5

# Plot trajectory (Form: Pixel_coordinates = a_* Cartesian + b_)
a_x: 3658.868971
b_x: 493
a_y: -3468.089952
b_y: 591.621936

# Motor values to finger angles
a_left: -0.002208
b_left: 1.038487
a_right: 0.002177
b_right: 0.307405

hold_left: 0.40
hold_right: 0.53

# Topics To be recorded
topics: {
  
  topic2: "object_trajectory",
  topic3: "object_position",
  topic4: "Finger_motor_position",
  topic5: "Action"
}
